### PR TITLE
[13.0] [MIG] sale_manual_delivery

### PR DESCRIPTION
**Pull request by phuctranfxvn**
Migrated from https://github.com/OCA/sale-workflow/pull/987

### CODE REVIEW

**Comment by newtratip** [comment]
@rousseldenis Could you change this milestone from 14.0 to 13.0 please ?

File: sale_manual_delivery/models/sale_order.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        string="Manual Delivery",
+        default=False,
+        help="If Manual, the deliveries are not created at SO confirmation.\
+        You need to use the Create Delivery button in order to reserve and \
+        ship the goods.",
+    )
+
+    @api.onchange("team_id")
+    def _onchange_team_id(self):
+        for so in self:
```
Comment (i-vyshnevska) [inline sale_manual_delivery/models/sale_order.py]: onchange runs on a single record no need to iterate here

File: sale_manual_delivery/models/stock_move.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
+# Copyright 2017 Denis Leemann, Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class StockMove(models.Model):
+    _inherit = "stock.move"
+
+    carrier_id = fields.Many2one("delivery.carrier", string="Delivery Method")
```
Comment (i-vyshnevska) [inline sale_manual_delivery/models/stock_move.py:10]: I think it is better to have the carrier on stock.picking level, I don't know cases when moves belong to one picking and served by different carriers

Comment (phuctranfxvn) [reply sale_manual_delivery/models/stock_move.py:10]: Hi @i-vyshnevska
Agree with you. However, this module is migrated from https://github.com/OCA/sale-workflow/pull/987. I think it is better to have this reviewed in mentioned PR first. I'll notify this and tag you to follow up

File: sale_manual_delivery/models/sale_order.py (hunk 1 above)
Comment (phuctranfxvn) [reply sale_manual_delivery/models/sale_order.py]: fixed in https://github.com/OCA/sale-workflow/pull/1319/commits/c31db4ce252ff937e9014fb4d5626e4116b5e5fe

**Comment by rousseldenis** [comment]
@phuctranerp Do you mind if we switch to #1566 ?

**Comment by phuctranfxvn** [comment]
> @phuctranerp Do you mind if we switch to #1566 ?

@rousseldenis Yes, that's fine for me, thanks

**Pull request closed without merging**